langchain/src/callbacks/base.ts

```diff
@@ -176,6 +176,12 @@ export abstract class BaseCallbackHandler
 
   ignoreAgent = false;
 
+  awaitHandlers =
+    typeof process !== "undefined"
+      ? // eslint-disable-next-line no-process-env
+        process.env?.LANGCHAIN_CALLBACKS_BACKGROUND !== "true"
```

Comment:
How about `LANGCHAIN_CALLBACK_MODE="async"` or `LANGCHAIN_CALLBACK_MODE="nonblocking"`?

Reply:
@agola11 do you prefer that?